controller.go

```diff
@@ -138,13 +153,13 @@ func (c *controller) Run(stopCh chan struct{}) {
 	defer utilruntime.HandleCrash()
 
 	// Let the workers stop when we are done
-	glog.Info("Starting kvm-operator node controller")
+	c.logger.Log("Info", "Starting kvm-operator node controller")
```

Comment:
We have `"info"` everywhere else. We also start message with lower letter.
BTW at some point we should add `Info, Warn, Error, Debug` methods I think.